packages/connect-query/vite.config.ts

```diff
@@ -12,15 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import type { Config } from "jest";
+import { defineConfig } from "vitest/config";
 
-const config: Config = {
-  preset: "../../jest-preset.js",
-  testEnvironment: "@bufbuild/jest-environment-jsdom",
-  moduleNameMapper: {
-    "(.+)\\.js": "$1", // https://connectrpc.com/docs/web/supported-browsers-and-frameworks/#jest
+// https://vitejs.dev/config/
+export default defineConfig({
+  test: {
+    environment: "jsdom",
```

Comment:
Wow, did jest simply fix their environment? 

https://github.com/vitest-dev/vitest/blob/e6730909b34526eb73b744124f76151624782679/packages/vitest/src/integrations/env/jsdom.ts#L88

Reply:
jest did so much stuff automatically. Eventually, we might take advantage of vitest's browser mode to actually run in the browser (like jasmine... software development is just a circle) and we can stop patching the dom, but for our purposes, this is fine.

Reply:
jest-environment-jsdom provides a DOM so that tests for web apps can run on Node.js. Awesome.

Unfortunately, it also removes the text encoding API from globals, which Node.js has been supporting for many years now. The problem is so bad that it forced us to create `@bufbuild/jest-environment-jsdom`. It's a simple fix, but Jest has been sitting the problem out for years now, blaming jsdom.

Vitest does the same thing with environments, but it fixes the problem (see link above). Kudos to Vitest.

Reply:
Ah I understand what you meant now